### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by s2504s**
## What
* Updated README

## Why
* Added sectiom `Help`

### CODE REVIEW

**Comment by drama17** [comment]
376898f0bc
 This should looks like:
```
| `security_group_ids` | List of IDs of AWS Security Groups associated with creating instance |
```

File: README.md [hunk 1]
```diff
@@ -98,3 +98,83 @@ resource "aws_ami_from_instance" "example" {
 
 ## References
 * Thanks to https://github.com/cloudposse/tf_bastion for the inspiration
```
Comment (osterman) [inline README.md]: fix repo name `tf_bastion`

Comment (osterman) [reply README.md]: It's also kind of strange to say "thanks" to ourselves. Let's just link to the repo, but drop the praise =)

**Comment by comeanother** [comment]
Build fails because of `fmt` issues in external modules
image

**Pull request merged**